PR#1647 – Liquid Exceptions

- [x] Liquid Exception: Could not find document '_posts/2025/06/2025-06-27-20250627ClosingIndexes.md' in tag 'link'. Make sure the document exists and the path is correct. in /home/runner/work/blog/blog/_posts/2025/06/2025-06-26-20250626ClosingIndexes.md

bundler: failed to load command: jekyll (/home/runner/work/blog/blog/vendor/bundle/ruby/3.3.0/bin/jekyll)
/home/runner/work/blog/blog/vendor/bundle/ruby/3.3.0/gems/jekyll-4.4.1/lib/jekyll/tags/link.rb:32:in `render': Could not find document '_posts/2025/06/2025-06-27-20250627ClosingIndexes.md' in tag 'link'. (ArgumentError)

```
Run bundle exec jekyll build --trace --incremental --baseurl "/blog"
  bundle exec jekyll build --trace --incremental --baseurl "/blog"
  shell: /usr/bin/bash -e {0}
  env:
    GITHUB_PAGES: true
    JEKYLL_ENV: production
    JEKYLL_GITHUB_TOKEN: 
    LOG_LEVEL: debug
  
To use retry middleware with Faraday v2.0+, install `faraday-retry` gem
Configuration file: /home/runner/work/blog/blog/_config.yml
            Source: /home/runner/work/blog/blog
       Destination: /home/runner/work/blog/blog/_site
 Incremental build: enabled
      Generating... 
  AI Related Posts: Creating cache [.ai_related_posts_cache.sqlite3]
  AI Related Posts: Fetch disabled. Using cached related posts data.
  AI Related Posts: Found 0 cached embeddings; skipped 848 fetches
       Jekyll Feed: Generating feed for posts
         AutoPages: Disabled/Not configured in site.config.
        Pagination: ----------------------------
        Pagination: Page: blogpager.html
        Pagination:  Active configuration
        Pagination:   Enabled:          true
        Pagination:   Items per page:   15
        Pagination:   Permalink:        /page/:num/
        Pagination:   Title:            :title - page :num of :max
        Pagination:   Limit:            0
        Pagination:   Sort by:          date
        Pagination:   Sort reverse:     true
        Pagination:  Active Filters
        Pagination:   Collection:       posts
        Pagination:   Offset:           0
        Pagination:   Category:         [Not set]
        Pagination:   Tag:              [Not set]
        Pagination:   Locale:           [Not set]
        Pagination:  Filtering by: Category  848 => 848
        Pagination:  Filtering by: Tag       848 => 848
        Pagination:  Filtering by: Locale    848 => 848
        Pagination: Rolling through the date fields for all documents
        Pagination: Complete, processed 1 pagination page(s)
DEPRECATION WARNING: to_time will always preserve the timezone offset of the receiver in Rails 8.0. To opt in to the new behavior, set `ActiveSupport.to_time_preserves_timezone = true`. (called from load at /home/runner/work/blog/blog/vendor/bundle/ruby/3.3.0/bin/jekyll:25)
  Liquid Exception: Could not find document '_posts/2025/06/2025-06-27-20250627ClosingIndexes.md' in tag 'link'. Make sure the document exists and the path is correct. in /home/runner/work/blog/blog/_posts/2025/06/2025-06-26-20250626ClosingIndexes.md
bundler: failed to load command: jekyll (/home/runner/work/blog/blog/vendor/bundle/ruby/3.3.0/bin/jekyll)
/home/runner/work/blog/blog/vendor/bundle/ruby/3.3.0/gems/jekyll-4.4.1/lib/jekyll/tags/link.rb:32:in `render': Could not find document '_posts/2025/06/2025-06-27-20250627ClosingIndexes.md' in tag 'link'. (ArgumentError)
Make sure the document exists and the path is correct.
        raise ArgumentError, <<~MSG
              ^^^^^^^^^^^^^^^^^^^^^
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.3.0/gems/liquid-4.0.4/lib/liquid/block_body.rb:103:in `render_node_to_output'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.3.0/gems/liquid-4.0.4/lib/liquid/block_body.rb:91:in `render'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.3.0/gems/liquid-4.0.4/lib/liquid/template.rb:206:in `block in render'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.3.0/gems/liquid-4.0.4/lib/liquid/template.rb:240:in `with_profiling'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.3.0/gems/liquid-4.0.4/lib/liquid/template.rb:205:in `render'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.3.0/gems/liquid-4.0.4/lib/liquid/template.rb:218:in `render!'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.3.0/gems/jekyll-4.4.1/lib/jekyll/liquid_renderer/file.rb:39:in `block (3 levels) in render!'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.3.0/gems/jekyll-4.4.1/lib/jekyll/liquid_renderer/file.rb:59:in `measure_counts'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.3.0/gems/jekyll-4.4.1/lib/jekyll/liquid_renderer/file.rb:38:in `block (2 levels) in render!'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.3.0/gems/jekyll-4.4.1/lib/jekyll/liquid_renderer/file.rb:63:in `measure_bytes'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.3.0/gems/jekyll-4.4.1/lib/jekyll/liquid_renderer/file.rb:37:in `block in render!'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.3.0/gems/jekyll-4.4.1/lib/jekyll/liquid_renderer/file.rb:70:in `measure_time'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.3.0/gems/jekyll-4.4.1/lib/jekyll/liquid_renderer/file.rb:36:in `render!'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.3.0/gems/jekyll-4.4.1/lib/jekyll/renderer.rb:129:in `render_liquid'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.3.0/gems/jekyll-4.4.1/lib/jekyll/renderer.rb:80:in `render_document'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.3.0/gems/jekyll-4.4.1/lib/jekyll/renderer.rb:63:in `run'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.3.0/gems/jekyll-4.4.1/lib/jekyll/site.rb:572:in `render_regenerated'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.3.0/gems/jekyll-4.4.1/lib/jekyll/site.rb:557:in `block (2 levels) in render_docs'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.3.0/gems/jekyll-4.4.1/lib/jekyll/site.rb:556:in `each'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.3.0/gems/jekyll-4.4.1/lib/jekyll/site.rb:556:in `block in render_docs'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.3.0/gems/jekyll-4.4.1/lib/jekyll/site.rb:555:in `each_value'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.3.0/gems/jekyll-4.4.1/lib/jekyll/site.rb:555:in `render_docs'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.3.0/gems/jekyll-4.4.1/lib/jekyll/site.rb:210:in `render'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.3.0/gems/jekyll-4.4.1/lib/jekyll/site.rb:80:in `process'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.3.0/gems/jekyll-4.4.1/lib/jekyll/command.rb:28:in `process_site'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.3.0/gems/jekyll-4.4.1/lib/jekyll/commands/build.rb:65:in `build'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.3.0/gems/jekyll-4.4.1/lib/jekyll/commands/build.rb:36:in `process'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.3.0/gems/jekyll-4.4.1/lib/jekyll/command.rb:91:in `block in process_with_graceful_fail'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.3.0/gems/jekyll-4.4.1/lib/jekyll/command.rb:91:in `each'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.3.0/gems/jekyll-4.4.1/lib/jekyll/command.rb:91:in `process_with_graceful_fail'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.3.0/gems/jekyll-4.4.1/lib/jekyll/commands/build.rb:18:in `block (2 levels) in init_with_program'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.3.0/gems/mercenary-0.4.0/lib/mercenary/command.rb:221:in `block in execute'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.3.0/gems/mercenary-0.4.0/lib/mercenary/command.rb:221:in `each'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.3.0/gems/mercenary-0.4.0/lib/mercenary/command.rb:221:in `execute'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.3.0/gems/mercenary-0.4.0/lib/mercenary/program.rb:44:in `go'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.3.0/gems/mercenary-0.4.0/lib/mercenary.rb:21:in `program'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.3.0/gems/jekyll-4.4.1/exe/jekyll:15:in `<top (required)>'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.3.0/bin/jekyll:25:in `load'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.3.0/bin/jekyll:25:in `<top (required)>'
	from /opt/hostedtoolcache/Ruby/3.3.8/x64/lib/ruby/3.3.0/bundler/cli/exec.rb:58:in `load'
	from /opt/hostedtoolcache/Ruby/3.3.8/x64/lib/ruby/3.3.0/bundler/cli/exec.rb:58:in `kernel_load'
	from /opt/hostedtoolcache/Ruby/3.3.8/x64/lib/ruby/3.3.0/bundler/cli/exec.rb:23:in `run'
	from /opt/hostedtoolcache/Ruby/3.3.8/x64/lib/ruby/3.3.0/bundler/cli.rb:455:in `exec'
	from /opt/hostedtoolcache/Ruby/3.3.8/x64/lib/ruby/3.3.0/bundler/vendor/thor/lib/thor/command.rb:28:in `run'
	from /opt/hostedtoolcache/Ruby/3.3.8/x64/lib/ruby/3.3.0/bundler/vendor/thor/lib/thor/invocation.rb:127:in `invoke_command'
	from /opt/hostedtoolcache/Ruby/3.3.8/x64/lib/ruby/3.3.0/bundler/vendor/thor/lib/thor.rb:527:in `dispatch'
	from /opt/hostedtoolcache/Ruby/3.3.8/x64/lib/ruby/3.3.0/bundler/cli.rb:35:in `dispatch'
	from /opt/hostedtoolcache/Ruby/3.3.8/x64/lib/ruby/3.3.0/bundler/vendor/thor/lib/thor/base.rb:584:in `start'
	from /opt/hostedtoolcache/Ruby/3.3.8/x64/lib/ruby/3.3.0/bundler/cli.rb:29:in `start'
	from /opt/hostedtoolcache/Ruby/3.3.8/x64/lib/ruby/gems/3.3.0/gems/bundler-2.5.22/exe/bundle:28:in `block in <top (required)>'
	from /opt/hostedtoolcache/Ruby/3.3.8/x64/lib/ruby/3.3.0/bundler/friendly_errors.rb:117:in `with_friendly_errors'
	from /opt/hostedtoolcache/Ruby/3.3.8/x64/lib/ruby/gems/3.3.0/gems/bundler-2.5.22/exe/bundle:20:in `<top (required)>'
	from /opt/hostedtoolcache/Ruby/3.3.8/x64/bin/bundle:25:in `load'
	from /opt/hostedtoolcache/Ruby/3.3.8/x64/bin/bundle:25:in `<main>'
Error: Process completed with exit code 1.
```

_Originally posted by @RalphHightower in https://github.com/RalphHightower/blog/issues/1647#issuecomment-3014780515_

--- a/_posts/2025/06/2025-06-27-20250627ClosingIndexes.md
+++ b/_posts/2025/06/2025-06-27-20250627ClosingIndexes.md
@@ -1,0 +1,309 @@
+---
+layout: post
+tags: [Constitution of the United States, Supreme Court of the United States (SCOTUS), US Courts, Federal Reserve Board, Jerome H. Powell, Department of Commerce (DOC), Treasury Department, Senate, House of Representatives, U.S. Department of the Treasury, Department of Commerce (DOC), President of the United States (POTUS), White House (WH), Trump crime businesses, Trump Organization, World Liberty Financial, $TRUMP, $MELANIA, The Mar-a-Lago Club, Trump International Golf Club, Trump National Doral Golf Club, Trump National Jupiter Golf Club, Trump National Golf Club Washington D.C., Trump National Golf Club Bedminster, Trump National Golf Club Colts Neck, Trump National Golf Club Philadelphia, Trump National Golf Club Hudson Valley, Trump National Golf Club Westchester, Trump National Golf Club Los Angeles, Trump International Golf Club Dubai, Trump International Golf Links & Hotel Ireland Doonbeg, Trump MacLeod House & Lodge Scotland, Trump Turnberry, Trump crime family, Donald J Trump, Eric F. Trump / LinkedIn, Donald Trump Jr. / LinkedIn, Ivanka Trump, Jared Kushner, Howard Lutnick, Howard W. Lutnick, Scott Bessent, Implementing the General Terms of The United States of America-United Kingdom Economic Prosperity Deal. Presidential Actions Executive Orders June 16 2025, Fact Sheet –  Implementing the General Terms of the U.S.-UK Economic Prosperity Deal. Fact Sheets. June 17 2025, Fact Sheet –  President Donald J. Trump Increases Section 232 Tariffs on Steel and Aluminum. Fact Sheets. June 3 2025, Adjusting Imports of Aluminum and Steel into the United States. Proclamations. June 3 2025, Modifying Reciprocal Tariff Rates to Reflect Discussions with the People’s Republic of China. Presidential Actions Executive Orders May 12 2025, Addressing Certain Tariffs on Imported Articles. Presidential Actions Executive Orders. April 29 2025, Amendments to Adjusting Imports of Automobiles and Automobile Parts Into the United States. Presidential Actions Proclamations. April 29 2025, Fact Sheet –  President Donald J. Trump Incentivizes Domestic Automobile Production. Fact Sheets. April 29 2025, Ensuring National Security and Economic Resilience Through Section 232 Actions on Processed Critical Minerals and Derivative Products. Presidential Actions Executive Orders. April 15 2025, Fact Sheet –  President Donald J. Trump Ensures National Security and Economic Resilience Through Section 232 Actions on Processed Critical Minerals and Derivative Products. Fact Sheets. April 15 2025, Clarification of Exceptions Under Executive Order 14257 of April 2 2025 as Amended – The White House. Presidential Actions Presidential Memoranda April 11 2025, Modifying Reciprocal Tariff Rates to Reflect Trading Partner Retaliation and Alignment. Presidential Actions Executive Orders April 9 2025, Amendment to Reciprocal Tariffs and Updated Duties as Applied to Low-Value Imports from the People’s Republic of China. Presidential Actions Executive Orders April 8 2025, Report to the President on the America First Trade Policy Executive Summary. Fact Sheets April 3 2025, Regulating Imports with a Reciprocal Tariff to Rectify Trade Practices that Contribute to Large and Persistent Annual United States Goods Trade Deficits. Presidential Actions Executive Orders April 2 2025, Further Amendment to Duties Addressing the Synthetic Opioid Supply Chain in the People’s Republic of China as Applied to Low-Value Imports. Presidential Actions Executive Orders April 2 2025, Fact Sheet –  President Donald J. Trump Declares National Emergency to Increase our Competitive Edge Protect our Sovereignty and Strengthen our National and Economic Security. Fact Sheets April 2 2025, Regulating Imports with a Reciprocal Tariff to Rectify Trade Practices that Contribute to Large and Persistent Annual United States Goods Trade Deficits. Presidential Actions Executive Orders April 2 2025, Fact Sheet –  President Donald J. Trump Closes De Minimis Exemptions to Combat China’s Role in America’s Synthetic Opioid Crisis. Fact Sheets April 2 2025, Further Amendment to Duties Addressing the Synthetic Opioid Supply Chain in the People’s Republic of China as Applied to Low-Value Imports. Presidential Actions Executive Orders April 2 2025, Fact Sheet –  President Donald J. Trump Adjusts Imports of Automobiles and Automobile Parts into the United States. Fact Sheets March 26 2025, The Staggering Cost of the Illicit Opioid Epidemic in the United States. Articles March 26 2025, Fact Sheet –  President Donald J. Trump Imposes Tariffs on Countries Importing Venezuelan Oil. Fact Sheets March 25 2025, Imposing Tariffs on Countries Importing Venezuelan Oil. Presidential Actions Executive Orders March 24 2025, More Investment More Jobs and More Money in Americans’ Pockets. Articles March 24 2025, President Trump Positions U.S. as Global Superpower in Manufacturing. Articles March 20 2025, President Trump is Remaking America into a Manufacturing Superpower. Articles March 12 2025, Amendment to Duties to Address the Flow of Illicit Drugs Across Our Southern Border. Presidential Actions March 6 2025, Amendment to Duties to Address the Flow of Illicit Drugs Across Our Northern Border. Presidential Actions March 6 2025, President Trump is Putting American Workers First — And Bringing Back American Manufacturing. Articles March 4 2025, President Trump is Securing Our Homeland. Articles March 4 2025, Fact Sheet –  President Donald J. Trump Proceeds with Tariffs on Imports from Canada and Mexico. Fact Sheets March 3 2025, Further Amendment to Duties Addressing the Synthetic Opioid Supply Chain in the People’s Republic of China. Presidential Actions March 3 2025, Amendment to Duties to Address the Situation at our Southern Border. Presidential Actions March 2 2025, Fact Sheet –  President Donald J. Trump Addresses the Threat to National Security from Imports of Timber Lumber and their Derivative Products. Fact Sheets March 1 2025, Addressing the Threat to National Security from Imports of Timber Lumber. Presidential Actions March 1 2025, Addressing the Threat to National Security from Imports of Copper. Presidential Actions February 25 2025, Fact Sheet –  President Donald J. Trump Addresses the Threat to National Security from Imports of Copper. Fact Sheets February 25 2025, Defending American Companies and Innovators From Overseas Extortion and Unfair Fines and Penalties.. Presidential Actions February 21 2025, Fact Sheet –  President Donald J. Trump Issues Directive to Prevent the Unfair Exploitation of American Innovation. Fact Sheets February 21 2025, Remarks by President Trump at Republican Governors Association Meeting. Remarks February 20 2025, President Trump Demands Fair Reciprocal Trade. Articles February 13 2025, Fact Sheet –  President Donald J. Trump Announces “Fair and Reciprocal Plan” on Trade. Fact Sheets February 13 2025, Reciprocal Trade and Tariffs. Articles February 13 2025, Fact Sheet –  President Donald J. Trump Restores Section 232 Tariffs. Fact Sheets February 11 2025, Adjusting Imports of Aluminum into The United States. Presidential Actions February 11 2025, Adjusting Imports of Steel into The United States. Presidential Actions February 10 2025, Fact Sheet –  President Donald J. Trump Restores American Competitiveness and Security in FCPA Enforcement. Fact Sheets February 10 2025, Amendment to Duties Addressing the Synthetic Opioid Supply Chain in the People’s Republic of China. Presidential Actions February 5 2025, Progress on the Situation at Our Northern Border. Presidential Actions February 3 2025, Progress on the Situation at Our Southern Border. Presidential Actions February 3 2025, Imposing Duties to Address the Synthetic Opioid Supply Chain in the People’s Republic of China. Presidential Actions February 1 2025, Imposing Duties to Address the Flow of Illicit Drugs Across Our Northern Border. Presidential Actions February 1 2025, Fact Sheet –  President Donald J. Trump Imposes Tariffs on Imports from Canada Mexico and China. Fact Sheets February 1 2025, Imposing Duties to Address the Situation at Our Southern Border. Presidential Actions February 1 2025, America First Trade Policy. Presidential Actions January 20 2025, tariffs, politics, stupidity]
+categories: [world stock market indexes]
+date: 2025-06-27 04:00:58 PM
+#excerpt: ''
+#image: 'BASEURL/assets/blog/img/.png'
+#description:
+#permalink:
+title: "2025-06-27: World Stock Market Closing Indexes: Americas (Moderate Gains). Europe, Middle East, & Africa (Strong Gains). Asia Pacific (Mixed)."
+---
+
+
+- [2025-06-26: World Stock Market Closing Indexes: Americas (Strong Gains). Europe, Middle East, & Africa (Moderate Gains). Asia Pacific (Mixed).]({% link _posts/2025/06/2025-06-26-20250626ClosingIndexes.md %})
+
+## [Stock Market Indexes - Google Finance](https://www.google.com/finance/markets/indexes/)
+
+| Index | Closing Value | Gain/Loss | Percentage Change |
+|---|---:|---:|---:|
+| **Americas** | | | |
+| S&P 500 | 6,173.99 | +32.97 | :arrow_up: 0.54% |
+| Dow Jones Industrial Average | 43,821.27 | +434.43 | :arrow_up: 1.00% |
+| Nasdaq Composite | 20,275.82 | +107.91 | :arrow_up: 0.54% |
+| Russell 2000 Index | 2,168.71 | -3.40 | :arrow_down: 0.16% |
+| S&P/TSX Composite Index | 26,620.06 | -131.89 | :arrow_down: 0.49% |
+| IBOVESPA | 136,843.95 | -269.94 | :arrow_down: 0.20% |
+| NYSE Composite | 20,295.53 | +39.33 | :arrow_up: 0.19% |
+| Dow Jones Transportation Average | 15,499.54 | +168.22 | :arrow_up: 1.10% |
+| Dow Jones Utility Average | 1,047.66 | +5.34 | :arrow_up: 0.51% |
+| S&P Latin America 40 | 2,658.61 | +0.84 | :arrow_up: 0.032% |
+| VIX | 16.52 | -0.070 | :arrow_down: 0.42% |
+| Russell 1000 Index | 3,368.60 | +7.78 | :arrow_up: 0.23% |
+| Dow Jones U.S. Total Stock Market Index | 60,996.35 | +290.73 | :arrow_up: 0.48% |
+| Barron's 400 Index | 1,310.97 | +2.43 | :arrow_up: 0.19% |
+| NASDAQ Composite Total Return | 24,853.76 | +129.11 | :arrow_up: 0.52% |
+| Nasdaq-100 | 22,534.25 | +86.96 | :arrow_up: 0.39% |
+| NASDAQ-100 Total Return | 27,317.58 | +105.92 | :arrow_up: 0.39% |
+| NASDAQ Transportation Index | 6,336.85 | +59.78 | :arrow_up: 0.95% |
+| NASDAQ Biotechnology Index | 4,224.71 | -15.58 | :arrow_down: 0.37% |
+| Nasdaq Financial-100 | 7,125.45 | -18.07 | :arrow_down: 0.25% |
+| Nasdaq Bank | 4,328.57 | +5.50 | :arrow_up: 0.13% |
+| Nasdaq Insurance | 15,491.00 | +46.82 | :arrow_up: 0.31% |
+| Nasdaq Industrial | 12,142.27 | +119.25 | :arrow_up: 0.99% |
+| Nasdaq Computer | 18,932.37 | +76.96 | :arrow_up: 0.41% |
+| S&P 400 | 3,103.30 | +8.84 | :arrow_up: 0.29% |
+| S&P 600 | 1,337.28 | +3.66 | :arrow_up: 0.27% |
+| Russell 2500 Index | 949.96 | +0.0063 | :arrow_up: 0.00066% |
+| Russell 3000 Index | 3,498.93 | +7.33 | :arrow_up: 0.21% |
+| NYSE American Composite Index | 5,719.62 | -145.12 | :arrow_down: 2.47% |
+| NYSE Arca Biotechnology Index | 5,573.01 | -28.82 | :arrow_down: 0.51% |
+| NYSE Arca Pharmaceutical Index | 920.31 | -7.17 | :arrow_down: 0.77% |
+| KBW Nasdaq Bank Index | 138.80 | +0.26 | :arrow_up: 0.19% |
+| Philadelphia Gold and Silver Index | 200.89 | -7.51 | :arrow_down: 3.60% |
+| PHLX Oil Service Sector | 58.57 | -0.26 | :arrow_down: 0.44% |
+| PHLX Semiconductor Sector | 5,543.58 | +2.12 | :arrow_up: 0.038% |
+| Avg Pct Chg: 0.01% | gainers: 24 (68.57%) | losers: 11 (31.43%) | **Moderate Gains** |
+| **Europe, Middle East, and Africa** | | | |
+| DAX PERFORMANCE-INDEX | 24,033.22 | +383.92 | :arrow_up: 1.62% |
+| FTSE 100 Index | 8,798.91 | +63.31 | :arrow_up: 0.72% |
+| CAC 40 | 7,691.55 | +134.24 | :arrow_up: 1.78% |
+| IBEX 35 | 13,969.00 | +153.50 | :arrow_up: 1.11% |
+| EURO STOXX 50 | 5,325.64 | +81.61 | :arrow_up: 1.56% |
+| S&P EURO | 2,376.41 | +34.04 | :arrow_up: 1.45% |
+| TA-125 Index | 2,952.41 | -21.07 | :arrow_down: 0.71% |
+| AEX index | 920.14 | +9.45 | :arrow_up: 1.04% |
+| PSI-20 | 7,523.59 | +92.40 | :arrow_up: 1.24% |
+| WIG | 103,728.06 | +347.35 | :arrow_up: 0.34% |
+| Tadawul All-Share Index | 11,068.27 | +94.29 | :arrow_up: 0.86% |
+| Swiss Market Index | 11,980.38 | +100.38 | :arrow_up: 0.84% |
+| S&P Europe 350 | 2,200.41 | +24.97 | :arrow_up: 1.15% |
+| STOXX Europe 600 | 543.63 | +6.15 | :arrow_up: 1.14% |
+| FTSE AIM All-Share Index | 769.33 | +2.29 | :arrow_up: 0.30% |
+| FTSE All-Share Index | 4,793.13 | +36.80 | :arrow_up: 0.77% |
+| OMX Copenhagen 20 | 1,743.35 | +19.08 | :arrow_up: 1.11% |
+| OMX Copenhagen 25 | 1,723.20 | +15.22 | :arrow_up: 0.89% |
+| WIG20 | 2,815.66 | +2.57 | :arrow_up: 0.091% |
+| WIG30 | 3,660.25 | +3.87 | :arrow_up: 0.11% |
+| SBF 120 | 5,851.27 | +98.11 | :arrow_up: 1.71% |
+| CAC LARGE 60 | 8,313.39 | +142.15 | :arrow_up: 1.74% |
+| BIST 100 | 9,404.89 | +103.84 | :arrow_up: 1.12% |
+| BIST 30 | 10,485.43 | +152.43 | :arrow_up: 1.48% |
+| HDAX PERFORMANCE-INDEX | 12,585.30 | +188.16 | :arrow_up: 1.52% |
+| OMX Helsinki 25 | 4,797.70 | +59.15 | :arrow_up: 1.25% |
+| SPI 20 PR | 11,710.07 | +98.11 | :arrow_up: 0.84% |
+| Swiss Performance Index | 16,607.38 | +145.25 | :arrow_up: 0.88% |
+| AMX index | 907.00 | +12.91 | :arrow_up: 1.44% |
+| BEL 20 | 4,509.03 | +38.13 | :arrow_up: 0.85% |
+| PSI ALL-SHARE GR | 4,912.10 | +68.38 | :arrow_up: 1.41% |
+| Austrian Traded Index | 4,418.88 | +61.15 | :arrow_up: 1.40% |
+| OMX Stockholm 30 | 2,506.17 | +55.42 | :arrow_up: 2.26% |
+| OMX Stockholm Benchmark_GI | 1,956.85 | +44.51 | :arrow_up: 2.33% |
+| OMX Iceland All-Share PI | 2,066.05 | -3.39 | :arrow_down: 0.16% |
+| OMX Riga_GI | 892.90 | -2.20 | :arrow_down: 0.25% |
+| OMX Tallinn | 2,066.58 | +0.016 | :arrow_up: 0.00076% |
+| OMX Vilnius_GI | 1,197.53 | +2.61 | :arrow_up: 0.22% |
+| TA-35 Index | 2,904.71 | -23.64 | :arrow_down: 0.81% |
+| Avg Pct Chg: 0.94% | gainers: 35 (89.74%) | losers: 4 (10.26%) | **Strong Gains** |
+| **Asia Pacific** | | | |
+| Nikkei 225 | 40,150.79 | +566.21 | :arrow_up: 1.43% |
+| SSE Composite Index | 3,424.23 | -24.23 | :arrow_down: 0.70% |
+| Hang Seng Index | 24,284.15 | -41.25 | :arrow_down: 0.17% |
+| BSE SENSEX | 84,058.90 | +303.03 | :arrow_up: 0.36% |
+| NIFTY 50 | 25,637.80 | +88.80 | :arrow_up: 0.35% |
+| S&P Asia 50 | 6,450.52 | +34.76 | :arrow_up: 0.54% |
+| TAIEX | 22,580.08 | +87.74 | :arrow_up: 0.39% |
+| KOSPI | 3,055.94 | -23.62 | :arrow_down: 0.77% |
+| S&P/ASX 200 | 8,514.20 | -36.60 | :arrow_down: 0.43% |
+| S&P/NZX 50 Index | 12,583.59 | +103.54 | :arrow_up: 0.83% |
+| S&P BSE - 100 | 26,840.88 | +112.02 | :arrow_up: 0.42% |
+| TOPIX | 2,840.54 | +35.85 | :arrow_up: 1.28% |
+| A-share Index | 3,589.15 | -25.44 | :arrow_down: 0.70% |
+| B-share Index | 258.22 | -0.58 | :arrow_down: 0.22% |
+| SZSE Component Index | 10,378.55 | +35.07 | :arrow_up: 0.34% |
+| SSE 180 Index | 8,644.42 | -60.56 | :arrow_down: 0.70% |
+| SSE 50 Index | 2,707.57 | -30.90 | :arrow_down: 1.13% |
+| ChiNext | 2,124.34 | +9.92 | :arrow_up: 0.47% |
+| CSI 300 Index | 3,921.76 | -24.26 | :arrow_down: 0.61% |
+| SZSE Composite Index | 2,052.56 | +7.74 | :arrow_up: 0.38% |
+| S&P/HKEX GEM Index | 16.90 | -0.060 | :arrow_down: 0.35% |
+| S&P/HKEX Large Cap Index | 36,161.54 | -47.34 | :arrow_down: 0.13% |
+| Hang Seng China Enterprises Index | 8,762.47 | -41.36 | :arrow_down: 0.47% |
+| S&P BSE - 500 | 37,060.93 | +164.98 | :arrow_up: 0.45% |
+| S&P BSE - 200 | 11,614.10 | +50.56 | :arrow_up: 0.44% |
+| LQ45 | 770.58 | +10.32 | :arrow_up: 1.36% |
+| IDX Composite | 6,897.40 | +65.26 | :arrow_up: 0.96% |
+| FTSE ASEAN All-Share Index | 854.30 | -0.61 | :arrow_down: 0.071% |
+| S&P/ASX 300 | 8,447.30 | -33.60 | :arrow_down: 0.40% |
+| S&P/NZX 20 Index | 7,381.71 | +71.62 | :arrow_up: 0.98% |
+| S&P/NZX All Index | 1,814.22 | +14.49 | :arrow_up: 0.81% |
+| Avg Pct Chg: 0.16% | gainers: 17 (54.84%) | losers: 14 (45.16%) | **Mixed** |
+
+### List of some of the major stock markets around the world and their locations:
+
+| Stock Exchange | Location |
+|---------------|----------|
+| **[New York Stock Exchange (NYSE)](https://www.nyse.com/index)** | New York City, USA |
+| **[Nasdaq Stock Market](https://www.nasdaq.com/)** | New York City, USA |
+| **[東京証券取引 (TSE)](https://www.jpx.co.jp/)** | Tokyo, Japan |
+| **[上海证券交易所 (SSE)](https://sse.com.cn/)** | Shanghai, China |
+| **[香港聯合交易所 (HKEX)](https://www.hkex.com.hk/)** | Hong Kong, China |
+| **[London Stock Exchange (LSE)](https://www.londonstockexchange.com/)** | London, United Kingdom |
+| **[Euronext](https://www.euronext.com/)** | Amsterdam, Brussels, Dublin, Lisbon, Milan, Oslo, Paris |
+| **[Toronto Stock Exchange (TSX)](https://www.tmx.com/)** | Toronto, Canada |
+| **[नेशनल स्टॉक एक्सचेंज (NSE)](https://www.nseindia.com/)** | Mumbai, India |
+| **[बंबई स्टॉक एक्सचेंज (BSE)](https://www.bseindia.com/)** | Mumbai, India |
+| **[深圳证券交易所 (SZSE)](https://www.szse.cn/)** | Shenzhen, China |
+| **[السوق المالية السعودية (تداول)](https://www.saudiexchange.sa/)** | Riyadh, Saudi Arabia |
+| **[Australian Securities Exchange (ASX)](https://www.asx.com.au/)** | Sydney, Australia |
+| **[Deutsche Börse (Frankfurt Stock Exchange)](https://www.deutsche-boerse.com/dbg-en)** | Frankfurt, Germany |
+| **[SIX Swiss Exchange](https://www.six-group.com/)** | Zurich, Switzerland |
+| **[한국거래소 (KRX)](https://www.krx.co.kr/)** | Seoul, South Korea |
+| **[臺灣證券交易所 (TWSE)](https://www.twse.com.tw/)** | Taipei, Taiwan |
+| **[Johannesburg Stock Exchange (JSE)](https://www.jse.co.za/)** | Johannesburg, South Africa |
+| **[首页](https://www.bursamalaysia.com/)** | Kuala Lumpur, Malaysia |
+| **[ตลาดหลักทรัพย์แห่งประเทศไทย (SET)](https://www.set.or.th/)** | Bangkok, Thailand |
+| **[新加坡交易所 (SGX)](https://www.sgx.com/)** | Singapore |
+| **[Bolsa Mexicana de Valores (BMV)](https://www.bmv.com.mx/)** | Mexico City, Mexico |
+| **[Московская Биржа (MOEX)](https://www.moex.com/)** | Moscow, Russia |
+| **[A Bolsa do Brasil (B3)](https://www.b3.com.br/)** | São Paulo, Brazil |
+
+### Where Can Trump Be Found?
+
+#### Where Trump Pretends To Work
+
+| Where Trump Pretends To Work |
+|---|
+| **[White House](https://www.whitehouse.gov)** |
+| 1600 Pennsylvania Ave., NW <br /> Washington, DC 20500 <br /> <a href="tel:+12024561111">+1 (202) 456-1111</a> (comments) <br /> <a href="tel:+12024561414">+1 (202) 456-1414</a> (switchboard) |
+
+#### If It’s The Weekend, Find Trump On One Of His Golf Courses
+
+| Where Trump Can Be Found On Weekends |
+|---|
+| **[Mar-a-Lago](https://www.maralagoclub.com/)** |
+| [The Mar-a-Lago Club](https://www.maralagoclub.com/) <br /> 1100 South Ocean Boulevard, <br /> Palm Beach, Florida 33480 <br /> <a href="tel+15618322600">+1 (561) 832-2600</a> |
+| **Florida, West Palm Beach** |
+| **[Trump International Golf Club](https://www.trumpinternationalpalmbeaches.com/)** <br /> 3505 Summit Blvd <br /> West Palm Beach FL 33406 <br /> <a href="tel:+15616820700">+1 (561) 682-0700</a> |
+| **California, Los Angeles** |
+| **[Trump National Golf Club Los Angeles](https://www.trumpnationallosangeles.com/)** <br /> One Trump National Drive <br /> Rancho Palos Verdes, CA 90275 <br /> <a href="tel:+13102655000">+1 (310) 265-5000</a> |
+| **Florida, Miami** |
+| **[Trump National Doral Golf Club](https://www.trumpgolfdoral.com/)** <br /> 4400 N.W. 87th Ave <br /> Miami, FL 33178 <br /> <a href="tel:+13055922000">+1 (305) 592-2000</a> |
+| **Florida, Jupiter** |
+| **[Trump National Jupiter Golf Club](https://www.trumpnationaljupiter.com/about)** <br /> 115 Eagle Tree Terrace <br /> Jupiter FL 33477 <br /> <a href="tel:+15616918700">+1 (561) 691-8700</a> |
+| **New Jersey, Bedminster** |
+| **[Trump National Golf Club Bedminster](https://www.trumpnationalbedminster.com/)** <br /> 900 Lamington Road <br /> Bedminster, NJ 07921 <br /> <a href="tel:+19084704400">+1 (908) 470-4400</a> |
+| **New Jersey, Colts Neck** |
+| **[Trump National Golf Club Colts Neck](https://www.trumpcoltsneck.com/)** <br /> One Trump National Blvd <br /> Colts Neck, NJ 07722 <br /> <a href="tel:+15619731550">+1 (561) 973-1550</a> |
+| **New Jersey, Pine Hill** |
+| **[Trump National Golf Club Philadelphia](https://www.trumpnationalphiladelphia.com/)** <br /> 500 West Branch Ave. <br /> Pine Hill, NJ 08021 <br /> <a href="tel:+18564353100">+1 (856) 435-3100</a> |
+| **New York, Hopewell Junction** |
+|  **[Trump National Golf Club Hudson Valley](https://www.trumpnationalhudsonvalley.com/)** <br /> 178 Stormville Rd <br /> Hopewell Junction, NY 12533 <br /> <a href="tel:+18452231600">+1 (845) 223-1600</a> |
+| **New York, Briarcliff Manor** |
+| **[Trump National Golf Club Westchester](https://www.trumpnationalwestchester.com/)** <br /> 100 Shadow Tree Lane <br /> Briarcliff Manor, NY 10510 <br /> <a href="tel:+19149440900">+1 (914) 944-0900</a> |
+| **North Carolina, Mooresville** |
+| **[Trump National Golf Club Charlotte](https://www.trumpnationalcharlotte.com/)** <br /> 120 Trump Square <br /> Mooresville, NC 28117 <br /> <a href="tel:+17047997300">+1 (704) 799-7300</a> |
+| **Virginia, Potomac Falls** |
+| **[Trump National Golf Club Washington, D.C.](https://www.trumpnationaldc.com/)** <br /> 20391 Lowes Island Blvd <br /> Potomac Falls, VA 20165 <br /> <a href="tel:+17034444801">+1 (703) 444-4801</a> |
+| **Dubai** |
+| **[Trump International Golf Club Dubai](https://www.trumpgolfdubai.com/)** <br /> DAMAC Hills <br /> Al Hebiah Third District Dubai <br /> <a href="tel:+971042453939">+971 (0) 4 245 3939</a> |
+| **Ireland, Clare** |
+| **[Trump International Golf Links & Hotel Ireland, Doonbeg](https://www.trumpgolfireland.com/)** <br />  DOONBEG CO. <br /> CLARE, IRELAND <br /> Phone: <a href="tel:+353659055600">+353 65 905 5600</a> |
+| **Scotland, Aberdeenshire** |
+| **[Trump MacLeod House & Lodge Scotland](https://www.trumphotels.com/macleod-house)** <br /> Menie Estate Balmedie <br /> Aberdeenshire, Scotland AB23 8YE <br /> PHONE: <a href="tel:+441358743300">+44 1358 743300</a> <br /> RESERVE: <a href="tel:+441358743300">+44 1358 743300</a> |
+| **Scotland, Ayrshire** |
+| **[Trump Turnberry](https://www.turnberry.co.uk/)** <br /> Maidens Road KA26 9LT Turnberry <br /> Ayrshire Scotland <br /> <a href="tel:+441655331000">+44.1655.331.000</a> <br /> <a href="tel:+4401655334115">+44 01655 334115</a> |
+
+- [Constitution of the United States](https://constitution.congress.gov/)
+- [Supreme Court of the United States (SCOTUS)](https://www.supremecourt.gov/)
+- [US Courts](https://www.uscourts.gov/)
+- [Federal Reserve Board](https://www.federalreserve.gov/)
+- [Jerome H. Powell](https://www.federalreserve.gov/aboutthefed/bios/board/powell.htm)
+- [Department of Commerce (DOC)](https://www.commerce.gov/)
+- [Treasury Department](https://home.treasury.gov/)
+- [Senate](https://www.senate.gov/)
+- [House of Representatives](https://www.house.gov/)
+- [U.S. Department of the Treasury](https://home.treasury.gov/)
+- [Department of Commerce (DOC)](https://www.commerce.gov/)
+- [President of the United States (POTUS)](https://www.whitehouse.gov/)
+- [White House (WH)](https://www.whitehouse.gov/)
+- Trump crime businesses 
+- [Trump Organization](https://www.trump.com/)
+- [World Liberty Financial](https://www.worldlibertyfinancial.com/)
+- [$TRUMP](https://gettrumpmemes.com/)
+- [$MELANIA](https://melaniameme.com/)
+- [The Mar-a-Lago Club](https://www.maralagoclub.com/) 
+- [Trump International Golf Club](https://www.trumpinternationalpalmbeaches.com/) 
+- [Trump National Doral Golf Club](https://www.trumpgolfdoral.com/) 
+- [Trump National Jupiter Golf Club](https://www.trumpnationaljupiter.com/) 
+- [Trump National Golf Club Washington, D.C.](https://www.trumpnationaldc.com/)
+- [Trump National Golf Club Bedminster](https://www.trumpnationalbedminster.com/) 
+- [Trump National Golf Club Colts Neck](https://www.trumpcoltsneck.com/) 
+- [Trump National Golf Club Philadelphia](https://www.trumpnationalphiladelphia.com/) 
+- [Trump National Golf Club Hudson Valley](https://www.trumpnationalhudsonvalley.com/) 
+- [Trump National Golf Club Westchester](https://www.trumpnationalwestchester.com/) 
+- [Trump National Golf Club Los Angeles](https://www.trumpnationallosangeles.com/) 
+- [Trump International Golf Club Dubai](https://www.trumpgolfdubai.com/) 
+- [Trump International Golf Links & Hotel Ireland, Doonbeg](https://www.trumpgolfireland.com/) 
+- [Trump MacLeod House & Lodge Scotland](https://www.trumphotels.com/macleod-house)
+- [Trump Turnberry](https://www.turnberry.co.uk/)
+- Trump crime family
+- [Donald J Trump](https://www.donaldjtrump.com/)
+- [Eric F. Trump / LinkedIn](https://www.linkedin.com/in/erictrump/)
+- [Donald Trump Jr. / LinkedIn](https://www.linkedin.com/in/donald-trump-jr-4454b862/)
+- Ivanka Trump
+- Jared Kushner
+- [Howard Lutnick](https://www.commerce.gov/about/leadership/howard-lutnick)
+- [Howard W. Lutnick](https://www.linkedin.com/in/howardwlutnick/)
+- [Scott Bessent](https://home.treasury.gov/about/general-information/officials/scott-bessent)
+- [Implementing the General Terms of The United States of America-United Kingdom Economic Prosperity Deal](https://www.whitehouse.gov/presidential-actions/2025/06/implementing-the-general-terms-of-the-united-states-of-america-united-kingdom-economic-prosperity-deal/). Presidential Actions, Executive Orders June 16, 2025
+- [Fact Sheet: Implementing the General Terms of the U.S.-UK Economic Prosperity Deal](https://www.whitehouse.gov/fact-sheets/2025/06/fact-sheet-implementing-the-general-terms-of-the-u-s-uk-economic-prosperity-deal/). Fact Sheets. June 17, 2025
+- [Fact Sheet: President Donald J. Trump Increases Section 232 Tariffs on Steel and Aluminum](https://www.whitehouse.gov/fact-sheets/2025/06/fact-sheet-president-donald-j-trump-increases-section-232-tariffs-on-steel-and-aluminum/). Fact Sheets. June 3, 2025
+- [Adjusting Imports of Aluminum and Steel into the United States](https://www.whitehouse.gov/presidential-actions/2025/06/adjusting-imports-of-aluminum-and-steel-into-the-united-states/). Proclamations. June 3, 2025
+- [Modifying Reciprocal Tariff Rates to Reflect Discussions with the People’s Republic of China](https://www.whitehouse.gov/presidential-actions/2025/05/modifying-reciprocal-tariff-rates-to-reflect-discussions-with-the-peoples-republic-of-china/). Presidential Actions, Executive Orders May 12, 2025
+- [Addressing Certain Tariffs on Imported Articles](https://www.whitehouse.gov/presidential-actions/2025/04/addressing-certain-tariffs-on-imported-articles/). Presidential Actions, Executive Orders. April 29, 2025
+- [Amendments to Adjusting Imports of Automobiles and Automobile Parts Into the United States](https://www.whitehouse.gov/presidential-actions/2025/04/amendments-to-adjusting-imports-of-automobiles-and-automobile-parts-into-the-united-states/). Presidential Actions, Proclamations. April 29, 2025
+- [Fact Sheet: President Donald J. Trump Incentivizes Domestic Automobile Production](https://www.whitehouse.gov/fact-sheets/2025/04/fact-sheet-president-donald-j-trump-incentivizes-domestic-automobile-production/). Fact Sheets. April 29, 2025
+- [Ensuring National Security and Economic Resilience Through Section 232 Actions on Processed Critical Minerals and Derivative Products](https://www.whitehouse.gov/presidential-actions/2025/04/ensuring-national-security-and-economic-resilience-through-section-232-actions-on-processed-critical-minerals-and-derivative-products/). Presidential Actions, Executive Orders. April 15, 2025
+- [Fact Sheet: President Donald J. Trump Ensures National Security and Economic Resilience Through Section 232 Actions on Processed Critical Minerals and Derivative Products](https://www.whitehouse.gov/fact-sheets/2025/04/fact-sheet-president-donald-j-trump-ensures-national-security-and-economic-resilience-through-section-232-actions-on-processed-critical-minerals-and-derivative-products/). Fact Sheets. April 15, 2025
+- [Clarification of Exceptions Under Executive Order 14257 of April 2, 2025, as Amended – The White House](https://www.whitehouse.gov/presidential-actions/2025/04/clarification-of-exceptions-under-executive-order-14257-of-april-2-2025-as-amended/). Presidential Actions, Presidential Memoranda April 11, 2025
+- [Modifying Reciprocal Tariff Rates to Reflect Trading Partner Retaliation and Alignment](https://www.whitehouse.gov/presidential-actions/2025/04/modifying-reciprocal-tariff-rates-to-reflect-trading-partner-retaliation-and-alignment/). Presidential Actions, Executive Orders April 9, 2025
+- [Amendment to Reciprocal Tariffs and Updated Duties as Applied to Low-Value Imports from the People’s Republic of China](https://www.whitehouse.gov/presidential-actions/2025/04/amendment-to-recipricol-tariffs-and-updated-duties-as-applied-to-low-value-imports-from-the-peoples-republic-of-china/). Presidential Actions, Executive Orders April 8, 2025
+- [Report to the President on the America First Trade Policy Executive Summary](https://www.whitehouse.gov/fact-sheets/2025/04/report-to-the-president-on-the-america-first-trade-policy-executive-summary/). Fact Sheets April 3, 2025
+- [Regulating Imports with a Reciprocal Tariff to Rectify Trade Practices that Contribute to Large and Persistent Annual United States Goods Trade Deficits](https://www.whitehouse.gov/presidential-actions/2025/04/regulating-imports-with-a-reciprocal-tariff-to-rectify-trade-practices-that-contribute-to-large-and-persistent-annual-united-states-goods-trade-deficits/). Presidential Actions, Executive Orders April 2, 2025
+- [Further Amendment to Duties Addressing the Synthetic Opioid Supply Chain in the People’s Republic of China as Applied to Low-Value Imports](https://www.whitehouse.gov/presidential-actions/2025/04/further-amendment-to-duties-addressing-the-synthetic-opioid-supply-chain-in-the-peoples-republic-of-china-as-applied-to-low-value-imports/). Presidential Actions, Executive Orders April 2, 2025
+- [Fact Sheet: President Donald J. Trump Declares National Emergency to Increase our Competitive Edge, Protect our Sovereignty, and Strengthen our National and Economic Security](https://www.whitehouse.gov/fact-sheets/2025/04/fact-sheet-president-donald-j-trump-declares-national-emergency-to-increase-our-competitive-edge-protect-our-sovereignty-and-strengthen-our-national-and-economic-security/). Fact Sheets April 2, 2025
+- [Regulating Imports with a Reciprocal Tariff to Rectify Trade Practices that Contribute to Large and Persistent Annual United States Goods Trade Deficits](https://www.whitehouse.gov/presidential-actions/2025/04/regulating-imports-with-a-reciprocal-tariff-to-rectify-trade-practices-that-contribute-to-large-and-persistent-annual-united-states-goods-trade-deficits/). Presidential Actions, Executive Orders April 2, 2025
+- [Fact Sheet: President Donald J. Trump Closes De Minimis Exemptions to Combat China’s Role in America’s Synthetic Opioid Crisis](https://www.whitehouse.gov/fact-sheets/2025/04/fact-sheet-president-donald-j-trump-closes-de-minimis-exemptions-to-combat-chinas-role-in-americas-synthetic-opioid-crisis/). Fact Sheets April 2, 2025
+- [Further Amendment to Duties Addressing the Synthetic Opioid Supply Chain in the People’s Republic of China as Applied to Low-Value Imports](https://www.whitehouse.gov/presidential-actions/2025/04/further-amendment-to-duties-addressing-the-synthetic-opioid-supply-chain-in-the-peoples-republic-of-china-as-applied-to-low-value-imports/). Presidential Actions, Executive Orders April 2, 2025
+- [Fact Sheet: President Donald J. Trump Adjusts Imports of Automobiles and Automobile Parts into the United States](https://www.whitehouse.gov/fact-sheets/2025/03/fact-sheet-president-donald-j-trump-adjusts-imports-of-automobiles-and-automobile-parts-into-the-united-states/). Fact Sheets March 26, 2025
+- [The Staggering Cost of the Illicit Opioid Epidemic in the United States](https://www.whitehouse.gov/articles/2025/03/the-staggering-cost-of-the-illicit-opioid-epidemic-in-the-united-states/). Articles March 26, 2025
+- [Fact Sheet: President Donald J. Trump Imposes Tariffs on Countries Importing Venezuelan Oil](https://www.whitehouse.gov/fact-sheets/2025/03/fact-sheet-president-donald-j-trump-imposes-tariffs-on-countries-importing-venezuelan-oil/). Fact Sheets March 25, 2025
+- [Imposing Tariffs on Countries Importing Venezuelan Oil](https://www.whitehouse.gov/presidential-actions/2025/03/imposing-tariffs-on-countries-importing-venezuelan-oil/). Presidential Actions, Executive Orders March 24, 2025
+- [More Investment, More Jobs, and More Money in Americans’ Pockets](https://www.whitehouse.gov/articles/2025/03/more-investment-more-jobs-and-more-money-in-americans-pockets/). Articles March 24, 2025
+- [President Trump Positions U.S. as Global Superpower in Manufacturing](https://www.whitehouse.gov/articles/2025/03/president-trump-positions-u-s-as-global-superpower-in-manufacturing/). Articles March 20, 2025
+- [President Trump is Remaking America into a Manufacturing Superpower](https://www.whitehouse.gov/articles/2025/03/president-trump-is-remaking-america-into-a-manufacturing-superpower/). Articles March 12, 2025
+- [Amendment to Duties to Address the Flow of Illicit Drugs Across Our Southern Border](https://www.whitehouse.gov/presidential-actions/2025/03/amendment-to-duties-to-address-the-flow-of-illicit-drugs-across-our-southern-border/). Presidential Actions March 6, 2025
+- [Amendment to Duties to Address the Flow of Illicit Drugs Across Our Northern Border](https://www.whitehouse.gov/presidential-actions/2025/03/amendment-to-duties-to-address-the-flow-of-illicit-drugs-across-our-northern-border-0c3c/). Presidential Actions March 6, 2025
+- [President Trump is Putting American Workers First — And Bringing Back American Manufacturing](https://www.whitehouse.gov/articles/2025/03/president-trump-is-putting-american-workers-first-and-bringing-back-american-manufacturing/). Articles March 4, 2025
+- [President Trump is Securing Our Homeland](https://www.whitehouse.gov/articles/2025/03/president-trump-is-securing-our-homeland/). Articles March 4, 2025
+- [Fact Sheet: President Donald J. Trump Proceeds with Tariffs on Imports from Canada and Mexico](https://www.whitehouse.gov/fact-sheets/2025/03/fact-sheet-president-donald-j-trump-proceeds-with-tariffs-on-imports-from-canada-and-mexico/). Fact Sheets March 3, 2025
+- [Further Amendment to Duties Addressing the Synthetic Opioid Supply Chain in the People’s Republic of China](https://www.whitehouse.gov/presidential-actions/2025/03/further-amendment-to-duties-addressing-the-synthetic-opioid-supply-chain-in-the-peoples-republic-of-china/). Presidential Actions March 3, 2025
+- [Amendment to Duties to Address the Situation at our Southern Border](https://www.whitehouse.gov/presidential-actions/2025/03/amendment-to-duties-to-address-the-situation-at-our-southern-border/). Presidential Actions March 2, 2025
+- [Fact Sheet: President Donald J. Trump Addresses the Threat to National Security from Imports of Timber, Lumber, and their Derivative Products](https://www.whitehouse.gov/fact-sheets/2025/03/fact-sheet-president-donald-j-trump-addresses-the-threat-to-national-security-from-imports-of-timber-lumber-and-their-derivative-products/). Fact Sheets March 1, 2025
+- [Addressing the Threat to National Security from Imports of Timber, Lumber](https://www.whitehouse.gov/presidential-actions/2025/03/addressing-the-threat-to-national-security-from-imports-of-timber-lumber/). Presidential Actions March 1, 2025
+- [Addressing the Threat to National Security from Imports of Copper](https://www.whitehouse.gov/presidential-actions/2025/02/addressing-the-threat-to-nationalsecurity-from-imports-of-copper/). Presidential Actions February 25, 2025
+- [Fact Sheet: President Donald J. Trump Addresses the Threat to National Security from Imports of Copper](https://www.whitehouse.gov/fact-sheets/2025/02/fact-sheet-president-donald-j-trump-addresses-the-threat-to-national-security-from-imports-of-copper/). Fact Sheets February 25, 2025
+- [Defending American Companies and Innovators From Overseas Extortion and Unfair Fines and Penalties](https://www.whitehouse.gov/presidential-actions/2025/02/defending-american-companies-and-innovators-from-overseas-extortion-and-unfair-fines-and-penalties/).. Presidential Actions February 21, 2025
+- [Fact Sheet: President Donald J. Trump Issues Directive to Prevent the Unfair Exploitation of American Innovation](https://www.whitehouse.gov/fact-sheets/2025/02/fact-sheet-president-donald-j-trump-issues-directive-to-prevent-the-unfair-exploitation-of-american-innovation/). Fact Sheets February 21, 2025
+- [Remarks by President Trump at Republican Governors Association Meeting](https://www.whitehouse.gov/remarks/2025/02/remarks-by-president-trump-at-republican-governors-association-meeting/). Remarks February 20, 2025
+- [President Trump Demands Fair, Reciprocal Trade](https://www.whitehouse.gov/articles/2025/02/president-trump-demands-fair-reciprocal-trade/). Articles February 13, 2025
+- [Fact Sheet: President Donald J. Trump Announces “Fair and Reciprocal Plan” on Trade](https://www.whitehouse.gov/fact-sheets/2025/02/fact-sheet-president-donald-j-trump-announces-fair-and-reciprocal-plan-on-trade/). Fact Sheets February 13, 2025
+- [Reciprocal Trade and Tariffs](https://www.whitehouse.gov/articles/2025/02/reciprocal-trade-and-tariffs/). Articles February 13, 2025
+- [Fact Sheet: President Donald J. Trump Restores Section 232 Tariffs](https://www.whitehouse.gov/fact-sheets/2025/02/fact-sheet-president-donald-j-trump-restores-section-232-tariffs/). Fact Sheets February 11, 2025
+- [Adjusting Imports of Aluminum into The United States](https://www.whitehouse.gov/presidential-actions/2025/02/adjusting-imports-of-aluminum-into-the-united-states/). Presidential Actions February 11, 2025
+- [Adjusting Imports of Steel into The United States](https://www.whitehouse.gov/presidential-actions/2025/02/adjusting-imports-of-steel-into-the-united-states/). Presidential Actions February 10, 2025
+- [Fact Sheet: President Donald J. Trump Restores American Competitiveness and Security in FCPA Enforcement](https://www.whitehouse.gov/fact-sheets/2025/02/fact-sheet-president-donald-j-trump-restores-american-competitiveness-and-security-in-fcpa-enforcement/). Fact Sheets February 10, 2025
+- [Amendment to Duties Addressing the Synthetic Opioid Supply Chain in the People’s Republic of China](https://www.whitehouse.gov/presidential-actions/2025/02/amendment-to-duties-addressing-the-synthetic-opioid-supply-chain-in-the-peoples-republic-of-china/). Presidential Actions February 5, 2025
+- [Progress on the Situation at Our Northern Border](https://www.whitehouse.gov/presidential-actions/2025/02/progress-on-the-situation-at-our-northern-border/). Presidential Actions February 3, 2025
+- [Progress on the Situation at Our Southern Border](https://www.whitehouse.gov/presidential-actions/2025/02/progress-on-the-situation-at-our-southern-border/). Presidential Actions February 3, 2025
+- [Imposing Duties to Address the Synthetic Opioid Supply Chain in the People’s Republic of China](https://www.whitehouse.gov/presidential-actions/2025/02/imposing-duties-to-address-the-synthetic-opioid-supply-chain-in-the-peoples-republic-of-china/). Presidential Actions February 1, 2025
+- [Imposing Duties to Address the Flow of Illicit Drugs Across Our Northern Border](https://www.whitehouse.gov/presidential-actions/2025/02/imposing-duties-to-address-the-flow-of-illicit-drugs-across-our-national-border/). Presidential Actions February 1, 2025
+- [Fact Sheet: President Donald J. Trump Imposes Tariffs on Imports from Canada, Mexico and China](https://www.whitehouse.gov/fact-sheets/2025/02/fact-sheet-president-donald-j-trump-imposes-tariffs-on-imports-from-canada-mexico-and-china/). Fact Sheets February 1, 2025
+- [Imposing Duties to Address the Situation at Our Southern Border](https://www.whitehouse.gov/presidential-actions/2025/02/imposing-duties-to-address-the-situation-at-our-southern-border/). Presidential Actions February 1, 2025
+- [America First Trade Policy](https://www.whitehouse.gov/presidential-actions/2025/01/america-first-trade-policy/). Presidential Actions January 20, 2025
+- tariffs
+- politics
+- stupidity
+
+

